cirq/google/api/v2/result.proto

```diff
@@ -16,6 +16,17 @@ message Result {
   repeated SweepResult sweep_results = 1;
 }
 
+// The result returned by a BatchedProgram
+message BatchedResult {
+
+  // The results of a BatchedProgram
+  // Each result in the message will directly correspond to a Program
+  // in the BatchedProgram request.  There will be one Result for each
```

Comment:
I think this should be slightly reworded, since there is a result per Program/RunContext pair.  I think it is important in the protos to convey the state of the resources, users may thinking about this as "run a program", but we should make it clear in the api that just creating a batched program does not ended up with results.

Reply:
Reworded.  Take another look and let me know if it is more clear.  I wasn't sure if I understood your concerns correctly.